src/parser/jobs/pld/changelog.tsx

```diff
@@ -1,10 +1,17 @@
 // import CONTRIBUTORS from 'data/CONTRIBUTORS'
 // import React from 'react'
 
+import React from 'react'
+
 export const changelog = [
 	// {
 	// 	date: new Date('2021-11-19'),
 	// 	Changes: () => <>The changes you made</>,
 	// 	contributors: [CONTRIBUTORS.YOU],
 	// },
+	{
+		date: new Date('2024-07-21'),
+		Changes: () => (<>Add rudimentary support for 7.0 and 7.01 Paladin</>),
+		contributors: [],
+	},
 ]
```

Comment:
I would add yourself as a Developer contributor so you get credit for your work!

Reply:
Done.